.github/workflows/lint-markdown-links-daily.yml

```diff
@@ -16,3 +16,23 @@ jobs:
           config-file: ".mlc.config.json"
           folder-path: "."
           max-depth: -1
+      - name: Send Notification to Slack
+        if: ${{ failure() }}
+        uses: slackapi/slack-github-action@v1.24.0
+        with:
+            payload: |
+              {
+                "text": "GitHub Action build result: FAILED\nDetails: ${{ github.event.pull_request.html_url || github.event.head_commit.url }}",
+                "blocks": [
+                  {
+                    "type": "section",
+                    "text": {
+                      "type": "mrkdwn",
+                      "text": "GitHub Action build result: FAILED\nDetails: ${{ github.event.pull_request.html_url || github.event.head_commit.url }}"
```

Comment:
Maybe it would be worth to add name of GH action?

Reply:
Reworked.